PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## Description
+
+What was changed in this pull request?
+
+Why is it necessary?
+
+Does it fix an existing issue? If so, please provide a link.
```

Comment:
```suggestion
Fixes #___
```

I think with a string like that github will auto-close the issue, and folks can just delete if there isn't one.